src/CloudGraphsDFG/entities/CloudGraphsDFG.jl

```diff
@@ -11,9 +11,9 @@ mutable struct CloudGraphsDFG{T <: AbstractParams} <: AbstractDFG
     robotId::String
     sessionId::String
     description::String
-    encodePackedTypeFunc
-    getPackedTypeFunc
-    decodePackedTypeFunc
+    encodePackedTypeFunc #TODO Is this planed to be used or deprecated?
+    getPackedTypeFunc #TODO Is this planed to be used or deprecated?
+    decodePackedTypeFunc #TODO Is this planed to be used or deprecated?
```

Comment:
My personal preference would be to use dispatch rather than hard wired function pointers.  I'm not sure if we'd be able to avoid the `"Packed"FactorType` approach which seems like a fair tradeoff.  If these fields are not used, then I'd say remove them ... although @GearsAD might have stronger recollection on why we had these and if they need to be kept.